src/components/BookCard/BookCard.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { BookCardWrapper } from './BookCard.styles';
+
+const BookCard = ({ title, author, cover }) => {
+  return (
+    <BookCardWrapper>
+      <img src={cover} alt={title} />
+      <h3>{title}</h3>
+      <span>by {author}</span>
+    </BookCardWrapper>
+  );
+};
+
+BookCard.propTypes = {
+  title: PropTypes.string.isRequired,
+  author: PropTypes.string.isRequired,
+  cover: PropTypes.string.isRequired
```

Comment:
using my limited knowledge, this seems fine, it's how I did the other TWO times I've used react...